che-base/scripts/base/commands/cmd_backup.sh

```diff
@@ -12,10 +12,10 @@
 cmd_backup() {
   debug $FUNCNAME
 
-  # possibility to skip che projects backup
+  # possibility to skip ${CHE_FORMAL_PRODUCT_NAME} projects backup
   SKIP_BACKUP_CHE_DATA=${1:-"--no-skip-data"}
   if [[ "${SKIP_BACKUP_CHE_DATA}" == "--skip-data" ]]; then
-    TAR_EXTRA_EXCLUDE="--exclude=instance/data/che"
+    TAR_EXTRA_EXCLUDE="--exclude=instance/data${CHE_CONTAINER_ROOT}"
```

Comment:
I don't know it's weird to me as well in other locations if we need to add / before this variable as it is already a path, so it may look odd to have a path not starting by /

like 
```
-v <local-path>:${CHE_CONTAINER_ROOT}"
```
using 
because in
```
-v <local-path>:/${CHE_CONTAINER_ROOT}"
```
it is like if we miss something in the variable